thirdeye/thirdeye-pinot/src/main/resources/validators/subscription/subscription-config-schema.json

```diff
@@ -0,0 +1,110 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Subscription Config Schema",
+
+  "type": "object",
+  "description": "YAML representation of a subscription group config",
+  "required": [ "subscriptionGroupName", "subscribedDetections", "application", "alertSchemes" ],
+
+  "properties": {
+    "subscriptionGroupName": {
+      "type": "string"
+    },
+    "application": {
+      "type": "string"
+    },
+    "type": {
+      "type": "string"
```

Comment:
shall the subscription group type be an enum type?

Reply:
replied above.